### ISSUE TITLE
Find the T.F.

### ISSUE THREAD

**Issue by vikaskumar5494** [vinaykumar2491/Project-Loop#13]
![github issues_6](https://cloud.githubusercontent.com/assets/12263067/12683201/f135967a-c6dd-11e5-9f21-8267bbfe0d5f.jpg)

**Comment by vikaskumar5494** [comment]
Snippet of solution
![github issues_5](https://cloud.githubusercontent.com/assets/12263067/12683233/256b8300-c6de-11e5-9a77-688ad30c1edc.jpg)

**Comment by vinaykumar2491** [comment]
the solution is correct.
Now tell me what does the horizontal and vertical axes represent in the above figure?
The horizontal axis does not represent 'time' b'coz the solution(poles & zeros) are solutions in s-domain not in t-domain.
If you disagree or still have doubts , comment below.

**Comment by vikaskumar5494** [comment]
Yes I do have doubt what horizontal axis represents? Vertical axis
represents imaginary part. Please clarify.

**Comment by vinaykumar2491** [comment]
the plot is in s-plane and is used to plot poles & zeros for a Transfer Function (in s-plane, or in Laplace plane).
**Note** that: the poles and zeros can be **purely Real** (s=a), **purely Imaginary**(s=jb), or **Complex**(s=a+jb).
So, **x-axis**  ==** real axis** (**_not time axis_**);
      _y-axis_  == _imaginary axis_.
real-axis has nothing to do with time. 

In the above question , the poles and zeros are all real and -ve  & hence the plots are on -ve x-axis.

**Comment by vikaskumar5494** [comment]
But the correct answer is (B)

**Comment by vinaykumar2491** [comment]
did u figure out why ur answer is wrong...
Also please solve question....make it like DPP, as we had discussed earlier if U solve 10Q/day it will be absolutely effective....choose whatever 10 questions u want (like 5 from Control, 5 from Aptitude...)
